apps/condo/pages/_app.tsx

```diff
@@ -62,59 +63,80 @@ const ANT_LOCALES = {
     en: enUS,
 }
 
-interface IMenuItemData {
-    path: string,
-    icon: React.FC,
-    label: string,
-}
-
-const ANT_DEFAULT_LOCALE = enUS
-
-const MenuItems: React.FC = () => {
-    const { link } = useOrganization()
-    const { isExpired } = useServiceSubscriptionContext()
-    const hasSubscriptionFeature = hasFeature('subscription')
-    const disabled = !link || (hasSubscriptionFeature && isExpired)
-    const { isCollapsed } = useLayoutContext()
-    const { wrapElementIntoNoOrganizationToolTip } = useNoOrganizationToolTip()
-
-    const menuItemsData: IMenuItemData[] = [{
+const itemsConfigs = [
+    {
         path: 'reports',
         icon: BarChartIconNew,
         label: 'menu.Analytics',
+        access: ['canGetExternalReportIframeUrl'],
     }, {
         path: 'ticket',
         icon: BarTicketIcon,
         label: 'menu.ControlRoom',
+        access: ['canReadTickets'],
     }, {
         path: 'property',
         icon: BarPropertyIcon,
         label: 'menu.Property',
+        access: ['canReadProperties'],
     }, {
         path: 'contact',
         icon: BarUserIcon,
         label: 'menu.Contacts',
+        access: ['canReadContacts'],
     }, {
         path: 'employee',
         icon: BarEmployeeIcon,
         label: 'menu.Employees',
+        access: ['canReadOrganizationEmployees'],
     }, {
         path: 'billing',
         icon: BarBillingIcon,
         label: 'menu.Billing',
+        access: ['canReadBillingReceipts'],
     }, {
         path: 'payments',
         icon: BarPaymentsIcon,
         label: 'menu.Payments',
+        access: ['canReadPayments'],
     }, {
         path: 'meter',
         icon: BarMeterIcon,
         label: 'menu.Meters',
+        access: ['canReadMeters'],
     }, {
         path: 'miniapps',
         icon: BarMiniAppsIcon,
         label: 'menu.MiniApps',
-    }]
+        access: ['canReadB2BApps'],
+    },
+]
+
+interface IMenuItemData {
+    path: string,
+    icon: React.FC,
+    label: string,
+    access: string[]
+}
+
+const ANT_DEFAULT_LOCALE = enUS
+
+const MenuItems: React.FC = () => {
+    const { link } = useOrganization()
+    const { isExpired } = useServiceSubscriptionContext()
+    const hasSubscriptionFeature = hasFeature('subscription')
+    const disabled = !link || (hasSubscriptionFeature && isExpired)
+    const { isCollapsed } = useLayoutContext()
+    const { wrapElementIntoNoOrganizationToolTip } = useNoOrganizationToolTip()
+    const role = get(link, 'role', {})
+
+    const menuItemsData: IMenuItemData[] = useMemo(() => {
+        const result = []
+        itemsConfigs.forEach((item) => {
+            if (get(role, `${get(item, 'access')}`, true)) result.push(item)
+        })
+        return result
```

Comment:
The template string isn't needed here :)
And looks like you forgot `item.access` is an array now.

Maybe something like this:
```suggestion
        return itemsConfigs.filter((item) => {
            const itemAccesses = get(item, 'access', [])
            return itemAccesses.length === 0 || itemAccesses.some((itemAccess) => !!role[itemAccess])
        })
```